packages/manager/modules/models/billing/Bill.class.js

```diff
@@ -0,0 +1,36 @@
+export default class Bill {
+  constructor({
+    billId,
+    category,
+    date,
+    debt,
+    pdfBetaUrl,
+    pdfUrl,
+    priceWithTax,
+    priceWithoutTax,
+    url,
+  }) {
+    Object.assign(this, {
+      billId,
+      category,
+      date,
+      pdfBetaUrl,
+      pdfUrl,
+      priceWithTax,
+      priceWithoutTax,
+      url,
+    });
+
+    this.debt = debt || {
```

Comment:
I don't get this, why is the default status `PAID` ? Also, shouldn't it be a constant so it can be reused ?

Reply:
If there are no debt in API, the bill is considered as paid. It could be a constant, though it has no other purpose that to be used in the model

Reply:
Maybe you could add a comment, because I don't think anyone can guess this just by reading your code :)

Reply:
I've added documentation on the newly created classes